routes/mandatarissen.ts

```diff
@@ -193,4 +194,22 @@ mandatarissenRouter.get(
   },
 );
 
+mandatarissenRouter.post('/download', async (req: Request, res: Response) => {
+  try {
+    const requestParameters = downloadMandatarissenUsecase.requestToJson(req);
+    const mandatarisData =
+      await downloadMandatarissenUsecase.fetchMandatarissen(requestParameters);
+    const csvString =
+      await downloadMandatarissenUsecase.transformToCsv(mandatarisData);
+
+    return res.status(STATUS_CODE.OK).send({ data: btoa(csvString) });
```

Comment:
Do we return the string or do we return a file id so the frontend can retrieve it from our file-service